auth-api/src/auth_api/models/user.py

```diff
@@ -44,6 +44,13 @@ class User(BaseModel):
     contacts = relationship('ContactLink', back_populates='user', primaryjoin='User.id == ContactLink.user_id')
     orgs = relationship('Membership', back_populates='user', primaryjoin='User.id == Membership.user_id')
 
+    is_terms_of_use_accepted = Column(Boolean(), default=False, nullable=True)
+    terms_of_use_accepted_version = Column(
+        ForeignKey('documents.version_id'), nullable=True
+    )
+    terms_of_use_version = relationship('Documents', foreign_keys=[terms_of_use_accepted_version], uselist=False,
```

Comment:
This would cause the document to be loaded every time on the user query which may have some performance impact as the document is TEXT, removing the  `terms_of_use_version = relationship('Documents', foreign_keys=[terms_of_use_accepted_version], uselist=False,` would be better I think.